### ISSUE TITLE
Is it possible to change pid for supporting F710 gamepad?

### ISSUE THREAD

**Issue by gyrorayo** [xurubin/logitech_unifying_pid_patch#8]
Hi, I found an old F710 gamepad without the adapter, getting a new one from online stores is expensive as hell and to make things worse I live in Mexico, so no shipping from the official Logitech store... But anyway, I have a Nano adapter with C52F PID, is it possible to change the PID to pair said gamepad using this adapter? (or any other logitech adapter)

I've read about this Solaar software and I found here:

https://github.com/pwr-Solaar/Solaar/blob/master/docs/usb.ids.txt

the ID for the F710 gamepad adapters (?)

c219  F710 Gamepad [DirectInput Mode]
c21f  F710 Wireless Gamepad [XInput Mode]

tldr; If I find a way to flash/replace the PID of my Nano adapter to one of the above IDs, would I be able to pair and use a device that is not a mouse or a keyboard? (a F710 gamepad in this very specific case)

thanks in advance.
